src/spikeinterface/sorters/launcher.py

```diff
@@ -68,10 +67,17 @@ def run_sorter_jobs(job_list, engine="loop", engine_kwargs={}, return_output=Fal
         The engine to run the list.
         * "loop" : a simple loop. This engine is
     engine_kwargs : dict
-
-    return_output : bool, dfault False
+        In the case of engine="slum", possible kwargs are:
+        - tmp_script_folder: str, default None
+            the folder in which the job scripts are created. Default: directory created by
+            the `tempfile` library
+        - sbatch_args: dict
+          arguments to be passed to sbatch. They are translated to the --args form.
```

Comment:
Maybe slightly reword to make a little clearer, that the only modification is the addition of `--`. There might be some confusion as previously it could take underscores for `cps_per_task` but I think this now must be `cpus-per-task`. So many something like 'arguments to be passed directly to sbatch. They will be automatically prefixed with `--`.  Arguments must be in the format slurm specify, see [documentation for sbatch]().